meinberlin/apps/plans/models.py

```diff
@@ -37,7 +37,14 @@ class Plan(UserGeneratedContentModel):
         (STATUS_DONE, _('done'))
     )
 
-    title = models.CharField(max_length=120, verbose_name=_('Title'))
+    title = models.CharField(max_length=120,
```

Comment:
It can stay like this now, but for next time, it would be nice if the formatting could be the same for every field. So as in the other field, every property should get it's own line and only the indentation matching that.